src/main/java/uk/gov/di/ipv/cri/common/library/domain/AuditEventTypes.java

```diff
@@ -1,7 +1,13 @@
 package uk.gov.di.ipv.cri.common.library.domain;
 
 public enum AuditEventTypes {
```

Comment:
Once Stu's PR has been merged this will need to be rebased and amended accordingly